submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionApiTest.kt

```diff
@@ -185,9 +186,61 @@ class SubmissionApiTest(
 
         val submitted = submissionRepository.getExtByAccNo("S-500")
 
-        assertThat(File("$ftpPath/${submitted.relPath}/Files").listFiles()?.size).isEqualTo(1)
-        assertThat(File("$ftpPath/${submitted.relPath}/Files/folder1")).exists()
-        assertThat(File("$ftpPath/${submitted.relPath}/Files/folder1")).isEmptyDirectory()
+        val ftpFiles = FileUtils.listAllFiles(File("$ftpPath/${submitted.relPath}/Files"))
+        val expectedFolder = File("$ftpPath/${submitted.relPath}/Files/folder1")
+        assertThat(ftpFiles).containsExactly(expectedFolder)
+        assertThat(expectedFolder).isEmptyDirectory()
+    }
+
+    @Test
+    @EnabledIfSystemProperty(named = "enableFire", matches = "false")
+    fun `16-8 submission released makes files public`() {
+        val submission = tsv {
+            line("Submission", "S-600")
+            line("Title", "Submission")
+            line("ReleaseDate", "2020-01-25")
+            line()
+
+            line("Study")
+            line()
+
+            line("File", "file_16-8.txt")
+            line()
+        }.toString()
+
+        webClient.uploadFile(tempFolder.createFile("file_16-8.txt", "16-8 file content"))
+        assertThat(webClient.submitSingle(submission, TSV)).isSuccessful()
+
+        val submitted = submissionRepository.getExtByAccNo("S-600")
+
+        val ftpFiles = FileUtils.listAllFiles(File("$ftpPath/${submitted.relPath}/Files"))
+        val expectedFile = File("$ftpPath/${submitted.relPath}/Files/file_16-8.txt")
+        assertThat(ftpFiles).containsExactly(expectedFile)
+        assertThat(expectedFile).hasContent("16-8 file content")
+    }
+
+    @Test
+    @EnabledIfSystemProperty(named = "enableFire", matches = "false")
+    fun `16-9 submission not released makes files private`() {
+        val submission = tsv {
+            line("Submission", "S-700")
+            line("Title", "Submission")
+            line("ReleaseDate", "2030-01-25")
+            line()
+
+            line("Study")
+            line()
+
+            line("File", "file_16-9.txt")
+            line()
+        }.toString()
+
+        webClient.uploadFile(tempFolder.createFile("file_16-9.txt", "16-9 file content"))
+        assertThat(webClient.submitSingle(submission, TSV)).isSuccessful()
+
+        val submitted = submissionRepository.getExtByAccNo("S-700")
+
+        assertThat(File("$ftpPath/${submitted.relPath}")).doesNotExist()
```

Comment:
we miss to check that file is present in private directory

Reply:
Ok!